Server/Dockerfile

```diff
@@ -11,13 +11,16 @@ ADD package*.json ./
 RUN npm install
 
```

Comment:
comment about:
# pull official base image
FROM node:13.12.0-alpine	FROM node:13.12.0-alpine
why not use node 14

Reply:
Updated